tests/manager_tests/schemas.py

```diff
@@ -58,6 +58,7 @@
     "updated": Or(None, str),
     "os": str,
     "first_reported": str,
+    "advisory_available": bool
```

Comment:
in this case, it would be good to add `Or(None, bool)` here

Reply:
added second attribute instead